### PR TITLE
Ut 3769 change Recorder to optional dependency + min Unity version 2019.4

### PR DESCRIPTION
**Pull request by vkovec**

### CODE REVIEW

File: com.unity.formats.fbx/Editor/Unity.Formats.Fbx.Editor.asmdef [hunk 1]
```diff
@@ -17,6 +17,12 @@
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.recorder",
+            "expression": "2.2.0-preview.4",
```
Comment (benoithudson) [inline com.unity.formats.fbx/Editor/Unity.Formats.Fbx.Editor.asmdef:23]: What's the interpretation of 'expression' -- is it a minimum version, or an exact match?

Comment (vkovec) [reply com.unity.formats.fbx/Editor/Unity.Formats.Fbx.Editor.asmdef:23]: it's minimum version, you can go higher

Comment (vkovec) [reply com.unity.formats.fbx/Editor/Unity.Formats.Fbx.Editor.asmdef:23]: for reference (under Version Defines section): https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html

File: com.unity.formats.fbx/Documentation~/recorder.md [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 With the FBX Exporter and the [Unity Recorder](https://docs.unity3d.com/Packages/com.unity.recorder@2.1/index.html), it is possible to export animations (including [Cinemachine](https://docs.unity3d.com/Packages/com.unity.cinemachine@2.3/manual/index.html) camera animations) directly to FBX in a few easy steps:
```
Comment (benoithudson) [inline com.unity.formats.fbx/Documentation~/recorder.md:3]: Should this be @2.2 for the recorder?

(Just checked and yes the version number is required)

**Review by benoithudson — COMMENTED** [review]
Update the link to the recorder to match the version we need and then please merge.

**Pull request merged**